apps/frontend/src/App.scss

```diff
@@ -54,20 +54,32 @@ $font-weight-black: 900;
 
 @font-face {
   font-family: $gellix-font-family;
-  src: url("./stories/assets/fonts/Gellix-Light_R.woff2") format("woff2");
+  src: url('./assets/fonts/Gellix-Light_R.woff2') format('woff2');
   font-weight: $font-weight-light;
 }
 
 @font-face {
   font-family: $gellix-font-family;
-  src: url("./stories/assets/fonts/Gellix-Regular_R.woff2") format("woff2");
+  src: url('./assets/fonts/Gellix-Regular_R.woff2') format('woff2');
   font-weight: $font-weight-regular;
   font-style: normal;
 }
 
 @font-face {
   font-family: $gellix-font-family;
-  src: url("./stories/assets/fonts/Gellix-SemiBold_R.woff2") format("woff2");
+  src: url('./assets/fonts/Gellix-SemiBold_R.woff2') format('woff2');
   font-weight: $font-weight-semibold;
   font-style: normal;
 }
+
+// BUTTON STYLES
+.btn-secondary {
```

Comment:
`App.scss` is a global style sheet for now. So please move button styles to `button.css` or `Button.scss` file. You can create folder `components/Button` and move both files there.